src/Node.hs

```diff
@@ -98,7 +100,7 @@ listenerMessageName (ListenerActionConversation f) =
                 -> Proxy rcv
                 -> MessageName
         msgName _ = messageName
-    in  msgName (f undefined) Proxy
+    in  msgName (f undefined undefined) Proxy
```

Comment:
Should be safe since `msgName`'s arguments is not used. @avieth if you consider it bad style, I'll remove these `undefined`s